deploy/crds/operators_v1_catalogsourceconfig_crd.yaml

```diff
@@ -44,3 +44,12 @@ spec:
             packages:
               type: string
               description: Comma separated list of operator(s) without spaces which will be enabled in the target namespace
+            csDisplayName:
+              type: string
+              description: DisplayName is passed along to the CatalogSource to be used as a pretty name.
+            csPublisher:
+              type: string
+              description: Publisher is passed along to the CatalogSource to be used to define what entity published the artifacts from the OperatorSource.
```

Comment:
Please replace `artifacts` with `operator bundles`

Reply:
A `bundle` is one `ClusterServiceVersion` and its resources. On the other hand, what we we are referring to here is operator `manifest` which contains a set of `bundle(s)` and information on `bundle to package` relationship. I think it's important to make this distinction.
I originally copied the description from the corresponding `go` file of each type to keep it consistent. I changed it to `operator manifest(s)`.